src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx

```diff
@@ -34,23 +34,23 @@ Complete the following steps to install the integration:
 
     1. Create a file named `nri-flex-nextcloud.yml` in `/etc/newrelic-infra/integrations.d`
 
-    2. Use our [configuration template](https://github.com/newrelic/nri-flex/blob/master/examples/json-read-cmd-example.yml) to update the fields `EVENT_TYPE` and `YOUR_DOMAIN` in the file named `nri-flex-nextcloud.yml`. The value on the `event_type` is used to store metrics on the NRDB. For example, you can update `EVENT_TYPE` to `NextcloudFlexMetrics`
+    2. Use our [configuration template](https://github.com/newrelic/nri-flex/blob/master/examples/json-read-cmd-example.yml) to update the fields `EVENT_TYPE` and `YOUR_DOMAIN` in the file named `nri-flex-nextcloud.yml`. The value on the `event_type` is used to store metrics in the NRDB. For example, you can update `EVENT_TYPE` to `NextcloudFlexMetrics`
```

Comment:
```suggestion
    2. Use our [configuration template](https://github.com/newrelic/nri-flex/blob/master/examples/json-read-cmd-example.yml) to update the fields `YOUR_EVENT_TYPE` and `YOUR_HOST_IP` in the file named `nri-flex-nextcloud.yml`. The value on the `event_type` is used to store metrics in the NRDB. For example, you can update `EVENT_TYPE` to `NextcloudFlexMetrics`
```

Reply:
@rhetoric101 You are correct. Updated the changes